google/cloud/pubsub/publisher_connection_test.cc

```diff
@@ -153,7 +155,9 @@ TEST(PublisherConnectionTest, FlowControl) {
   EXPECT_THAT(rejected.get(), StatusIs(StatusCode::kFailedPrecondition));
 
   publisher->Flush({});
-  publish.PopFront().set_value();
+  for (int i = 0; i < publish_count; i++) {
```

Comment:
Style nit, but JFYI: Prefer preincrement unless you explicitly need post-increment semantics.
https://google.github.io/styleguide/cppguide.html#Preincrement_and_Predecrement

Reply:
Fixed.